docs/docs/using-pants/key-concepts/targets-and-build-files.mdx

```diff
@@ -166,6 +166,45 @@ You can use the prefix `!!` to transitively exclude a dependency, meaning that e
 Transitive excludes can only be used in target types that conventionally are not depended upon by other targets, such as `pex_binary`, `python_distribution`, and `python_test` / `python_tests`. This is meant to limit confusion, as using `!!` in something like a `python_source` / `python_sources` target could result in surprising behavior for everything that depends on it. (Pants will print a helpful error when using `!!` when it's not legal.)
 :::
 
+## Referring to a group of targets
```

Comment:
I find these two sections so similar (same, but with different use cases) perhaps consider pushing them down a level?, so we could have:
```markdown
## Using the generic `target`

// brief introduction of the `target` target.

### Referring to a group of targets

[...]

### Creating aliases for targets

[...]
```